spec/sections/standard.adoc

```diff
@@ -682,8 +680,13 @@ in passing information to the executor† to differentiate between data and job
 |Required
 2+|MUST be an integer indicating the exit code of the executing job process.
 
-|`title`
+|`name`
 |Required
+2+|MUST be a string of only alphabetic, dash or underscore characters (defined by the regex† `^[a-zA-Z_-]+$`) 
+indicating the unique name to use for referring to this error.
```

Comment:
I would recommend a sentence here saying that a re-used error name from version to version could be used to indicate the same error across versions.